tests/yam/agama/agama_encrypted_lvm.pm

```diff
@@ -0,0 +1,34 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Run interactive installation with Agama,
+# run playwright tests directly from the Live ISO.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+
+use testapi qw(
+  assert_screen
+  assert_script_run
+  get_required_var
+  select_console
+  upload_logs
+);
+use power_action_utils 'power_action';
+use transactional 'process_reboot';
```

Comment:
this is not needed, hope we can get rid of repeating this tests over and over, we have a ticket in progress for that.